### DISCUSSION TITLE
Is is possible to enable IP Pools after installing Postal Server

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [postalserver/postal#2031] [Help with using Postal] [answered]
So, I enabled `use_ip_pools` config on `postal.yml` after I installed Postal Server. I then did `postal upgrade`, but I don't see the `IP Pools` link on the top right as the documentation says. Do I have to reinstall Postal again with the config enabled or is it just possible to enable them after having installed the server?

**Comment by willpower232** [comment, marked as answer]
You should only need to restart postal after you change the config file.

**Reply by abishekrsrikaanth** [reply]
Thanks, I will remember this in the future. I was hoping the restart would happen when you run `postal upgrade`. Appreciate the quick help.
